.github/workflows/_unit-tests-and-static-checks.yml

```diff
@@ -47,6 +51,12 @@ jobs:
           command: test
           args: -- --skip clique_network
 
+      - name: Install Nightly Rust Toolchain (for unstable features in tests)
+        uses: Cardinal-Cryptography/github-actions/install-rust-toolchain@v4
```

Comment:
One small feature request please: since `liminal` is behind the feature flag, it should be excluded from every commit to PR for now. Instead, let's add a label e.g. `liminal-CI` here https://github.com/Cardinal-Cryptography/aleph-node/blob/main/.github/workflows/on-pull-request-label.yml and spawn any liminal-related checks only if this label is added to the PR. 

On breaking liminal things e.g. by substrate updates: we should run the same liminal-checks in nightly, so then we can catch errors as a whole for every change in `aleph-node`

Reply:
discussed offline: I'll take care of this in the next PR tomorrow (so just not to mix scopes)

Reply:
ok so resolving this comment